### DISCUSSION TITLE
When will better(the proposal in the issue) documentation arrive?

### DISCUSSION THREAD

**Discussion by C-BJ** [hylo-lang/hylo#737] [Q&A]
When will better(the proposal in the issue) documentation arrive?

**Comment by kyouko-taiga** [comment]
Hello Cai,

What proposal are you referring to?

**Comment by C-BJ** [comment]
Suggestions for documentation in [issue](https://github.com/val-lang/val-lang.github.io/issues), like `Write "Val for Rust programmers"`

It just mean that when will there be more complete documentation

**Comment by kyouko-taiga** [comment]
We don't have specific timelines for these items yet, as we're currently focusing most of our efforts on the compiler implementation. Is there a particular topic that you would like to see developed?

**Comment by C-BJ** [comment]
OK, documentation is not a problem for now and Cppcon2022's Val video is due to go live. 
Maybe link video about Val on README to let more people know about the videoes?
btw: Why is github discussion in this place instead in val compiler's repository? Wouldn't that make easier to access(jump)?
@kyouko-taiga

**Comment by C-BJ** [comment]
I think the [RoadMap](https://www.val-lang.dev/pages/implementation-status.html) should be updated anyway @kyouko-taiga 

#48
